modules/shell/command.go

```diff
@@ -66,7 +66,8 @@ func RunCommandAndGetOutputE(t *testing.T, command Command) (string, error) {
 		return "", err
 	}
 
-	if err := cmd.Start(); err != nil {
+	err = cmd.Start()
+	if err != nil {
```

Comment:
I though the one-line `if err := xxx; err != nil { ... }` pattern was idiomatic Go?

Reply:
It is but it lead to `err` shadowing `err` from its parent scope. We can change it to

```go
if err = cmd.Start(); err != nil {
```

if you like that better. I think the version I changed it to is easier to read but that's a personal preference and I'm happy to change it back if that's what you prefer.

Reply:
It seems like in these cases, shadowing, rather than reusing the same variable name, is a good thing, no? It makes each error check standalone and you can never accidentally reuse an error from some previous call...